src/AutoMapper/QueryableExtensions/Impl/MemberGetterExpressionResultConverter.cs

```diff
@@ -21,7 +21,9 @@ public ExpressionResolutionResult GetExpressionResolutionResult(ExpressionResolu
         private static ExpressionResolutionResult ExpressionResolutionResult(
             ExpressionResolutionResult expressionResolutionResult, MemberInfo getter)
         {
-            var member = Expression.MakeMemberAccess(expressionResolutionResult.ResolutionExpression, getter);
+            var member = (getter is MethodInfo method && method.IsStatic && method.GetParameters().Count() == 1)
```

Comment:
The methods are already filtered, so just checking whether it's a method should be enough.